src/js/rx/components/PrescriptionTable.jsx

```diff
@@ -41,9 +41,9 @@ class PrescriptionTable extends React.Component {
         id: item.id,
         prescriptionName: (
           <div>
-            <Link to={`/${attrs.prescriptionId}`}>
+            <Link to={`/${attrs.prescriptionId}`} className="rx-prescription-name">
               {attrs.prescriptionName}
-            </Link><br/>
```

Comment:
The actual `TrackPackageLink` only uses the `usa-button` class. Would that suffice for this `Link`?